### ISSUE TITLE
AxeDevTools throws accessibility issues for grouped Grid

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#4534]
### Description

When the Grid is grouped, AxeDevTools throws an error for the rendered ChipList:

`role="none"` or `role="presentation"` removes the element from the accessibility tree, meaning assistive technologies (like screen readers) will ignore it completely. 
`aria-orientation="horizontal"` is an ARIA property that is meant to provide information about the element’s orientation. However, if the element has `role="none"`, it won't be exposed to assistive technologies, making `aria-orientation` and any other aria attributes useless.

**Ensure an element's role supports its ARIA attributes**

role='none' issue
 
<img width="960" alt="Image" src="https://github.com/user-attachments/assets/456365c3-3b1a-4278-afe0-b63fbfbf40e6" />

aria-selected='false' issue

<img width="962" alt="Image" src="https://github.com/user-attachments/assets/21d77f2a-6c5d-48d9-9cb3-c58e8dd70335" />

### Steps To Reproduce

1. Open the demo in a new window - https://www.telerik.com/kendo-angular-ui/components/grid/grouping/basics
1. Group the Grid by any field
1. Scan the page with AxeDevTools

### Actual Behavior

By default, the ChipList has role="listbox", but has been removed in the scope of the Grid to keep the button roles hierarchy.

`aria-selected` attribute on the Chip should be revised.

### Expected Behavior

When role="none" is applied no other accessibility attributes should be set.

**Comment by yanmariomenev** [comment]
The fix is officially available in version [19.3.0](https://www.telerik.com/kendo-angular-ui/components/changelogs/kendo-angular-ui#v19.3.0).
